hardhat.config.js

```diff
@@ -81,4 +81,10 @@ module.exports = {
   etherscan: {
     apiKey: process.env.ETHERSCAN_API_KEY,
   },
+  paths: {
+    sources: "./contracts",
+    tests: "./test",
+    cache: "./cache",
+    artifacts: "./client/src/artifacts"
```

Comment:
Is it possible to only override the `artifacts` key? The rest of the keys are using defaults values, so might be confusing the need for overriding them. Thoughts?

Reply:
Sure, np!